src/DataValues/ImportValue.php

```diff
@@ -70,12 +72,19 @@ class ImportValue extends DataValue {
 	 * @var string
 	 */
 	private $declarativeName = '';
+	
+	/** @var array */
+	private $declarativeNames = [];
+
+	/** @var mediaWikiNsContentReader */
+	private $mediaWikiNsContentReader;
```

Comment:
Type is wrong capitalization. Also should use PHP type, not a doc

`private ClassName $foobar;`